zika/zika.process.py

```diff
@@ -46,5 +46,4 @@
     runner.build_tree()
     runner.timetree_setup_filter_run()
     runner.run_geo_inference()
-    runner.save_as_nexus()
```

Comment:
i've reinstated this in 0301b642940cf64693fb839fc5ff82dcf629c946

There's now a script to turn the `tree.json` into a nexus file, with options such as temporal vs substitution branch lengths & the ability to resolve polytomies, which may be more useful to people.